src/main/java/de/allcom/exceptions/RestExceptionHandler.java

```diff
@@ -14,6 +20,36 @@ public ResponseEntity<StandardResponseDto> handleRestException(RestException e)
                 .status(e.getStatus())
                 .body(StandardResponseDto.builder()
                         .message(e.getMessage())
+                        .timestamp(LocalDateTime.now())
```

Comment:
delete this

Reply:
The problem is in class StandardResponseDto. You created this class with:
    @Schema(description = "The timestamp when the response is generated", example = "2023-11-21T15:30:00")
    private LocalDateTime timestamp;

Reply:
delete it there